nerdlandbot/commands/space_launches.py

```diff
@@ -111,6 +125,147 @@ def add_embed_field_for_upcominglaunch(self, index, result_json, embed):
+            )
+            return await ctx.send(translate("percy_tweet_error", await culture(ctx)))
+        return await ctx.send(last_status.text)
+
+    async def wait_for_tweet_reaction(self, ctx: commands.Context, msg_id: int,
+                                       timeout: int = REACTION_TIMEOUT):
+        end_time = time.time() + timeout
+        while True:
+            try:
+                reaction, user = await ctx.bot.wait_for(
+                    "reaction_add",
+                    check=lambda emoji, author: emoji.message.id == msg_id and not author.bot,
+                    timeout=timeout,
+                )
+                if reaction.emoji == bird:
+                    return await self.send_percy_tweet(ctx)
+            except asyncio.TimeoutError:
+                pass
+
+            if time.time() > end_time:
+                break
+
+    async def wait_for_camera_reaction(self, ctx: commands.Context, msg, embed, timeout: int = REACTION_TIMEOUT):
+        end_time = time.time() + timeout
+        while True:
+            try:
+                reaction, user = await ctx.bot.wait_for(
+                    "reaction_add",
+                    check=lambda emoji, author: emoji.message.id == msg.id and not author.bot,
+                    timeout=timeout,
+                )
+                percy_image = await self.get_percy_image()
+                if reaction.emoji == camera and percy_image:
+                    embed.set_image(url=percy_image)
+                    return await msg.edit(embed=embed)
+                elif reaction.emoji == camera:
+                    return await ctx.send(translate("percy_img_error", await culture(ctx)))
+            except asyncio.TimeoutError:
+                pass
+
+            if time.time() > end_time:
+                break
+
+    @commands.command(name="percy", hidden = False, help="percy_help", brief="percy_brief")
+    async def cmd_percy(self, ctx:commands.Context):
+        percy_data = await self.get_percy_data()
+
+        if percy_data:
+            embed=discord.Embed(title="Status update", 
+                                description=translate("percy_description", await culture(ctx)), 
+                                color=0xb33a00
+            )
+            embed.set_author(name="NASA Perseverance rover", 
+                            url="https://mars.nasa.gov/mars2020/", 
+                            icon_url=PERCY_ICON
+            )
+            
+            embed.add_field(name="Sol", value=percy_data['sol'], inline=False)
+            embed.add_field(name="Latitude", value=percy_data['latitude'], inline=True)
+            embed.add_field(name="Longitude", value=percy_data['longitude'], inline=True)
+            embed.add_field(name="Distance Driven", value=percy_data['distance'], inline=True)
+            if self.twitter_enabled:
+                embed.set_footer(text=translate("percy_footer_both", await culture(ctx)))
```

Comment:
Can we add the emoji's to the footer as well? This would make it clearer for me.